index.html

```diff
@@ -1,58 +1,385 @@
+              <div class="bg" id="card-work">
+                <div class="prjectDescription">
+                  <h3>
+                    Profesional Art <br />
+                    Printing Data
+                  </h3>
+                  <p>
+                    A daily selection of privately personalized reads; no
+                    accounts or sign-ups required. has been the industry's
+                    standard
+                  </p>
+                </div>
+                <ul>
+                  <li>html</li>
+                  <li>bootstrap</li>
+                  <li>Ruby</li>
+                </ul>
+              </div>
+              <button>See Project</button>
+            </div>
+
+            <div id="cardWork">
+              <div class="bg" id="card-work">
+                <div class="prjectDescription">
+                  <h3>
+                    Profesional Art <br />
+                    Printing Data
+                  </h3>
+                  <p>
+                    A daily selection of privately personalized reads; no
+                    accounts or sign-ups required. has been the industry's
+                    standard
+                  </p>
+                </div>
+                <ul>
+                  <li>html</li>
+                  <li>bootstrap</li>
+                  <li>Ruby</li>
+                </ul>
+              </div>
+              <button>See Project</button>
+            </div>
+
+            <div id="cardWork">
+              <div class="bg" id="card-work">
+                <div class="prjectDescription">
+                  <h3>
+                    Profesional Art <br />
+                    Printing Data
+                  </h3>
+                  <p>
+                    A daily selection of privately personalized reads; no
+                    accounts or sign-ups required. has been the industry's
+                    standard
+                  </p>
+                </div>
+                <ul>
+                  <li>html</li>
+                  <li>bootstrap</li>
+                  <li>Ruby</li>
+                </ul>
+              </div>
+              <button>See Project</button>
+            </div>
+
+            <div id="cardWork">
+              <div class="bg" id="card-work">
+                <div class="prjectDescription">
+                  <h3>
+                    Profesional Art <br />
+                    Printing Data
+                  </h3>
+                  <p>
+                    A daily selection of privately personalized reads; no
+                    accounts or sign-ups required. has been the industry's
+                    standard
+                  </p>
+                </div>
+                <ul>
+                  <li>html</li>
+                  <li>bootstrap</li>
+                  <li>Ruby</li>
+                </ul>
+              </div>
+              <button>See Project</button>
+            </div>
+          </section>
```

Comment:
- [ ] Your project cards are good, but there is no gap between the cards and they are not responsive making some cards disappear in smaller desktop window sizes. I recommend you not set a fixed width to style your cards and maybe use some `%` value for it. Also, please add some gaps between the cards. Kindly fix that.
 - [ ] The hover effect on the cards should make the title, description, dark filter, and tags disappear and the button 'See project' should appear. Currently, the data inside the card does not disappear and the button is always appearing. Kindly fix this to make it look like the Figma template.
 - Your Design:
 - 
<img width="1025" alt="image" src="https://user-images.githubusercontent.com/61919617/206696169-23b55258-9f0b-4d97-b675-b561ab2ef5f3.png">

 - Template:
 - 
<img width="794" alt="image" src="https://user-images.githubusercontent.com/61919617/206696245-d9fe79a7-f738-4d62-8852-57b9e1defd4e.png">

Reply:
You are right